src/transports/net.js

```diff
@@ -39,6 +39,9 @@ module.exports = class Connection extends EventEmitter {
             } else {
                 this.socket.write(line + '\r\n', cb);
             }
+        } else {
+            this.debugOut('writeLine() called when not connected');
+            cb();
```

Comment:
`cb()` should be called on the next tick so that it matches the `.write()` cb call, otherwise it can cause code execution in different orders.

Reply:
In `connection` too?

Will babel compile `process.nextTick` into a setTimeout or do I have to do an if check myself there?

Reply:
> In connection too?
Yup

> Will babel compile process.nextTick into a setTimeout or do I have to do an if check myself there?
I actually have no idea but `setTimeout` works as you've already got it there now